src/tests/running.md

```diff
@@ -213,6 +213,23 @@ This is much faster, but doesn't always work. For example, some tests
 include directives that specify specific compiler flags, or which rely
 on other crates, and they may not run the same without those options.
 
+## Running `run-make` tests
+
+### Windows
+
+Running the `run-make` test suite on Windows is a bit more involved. There are numerous
+prerequisites and environmental requirements:
+
+- Install python on Windows: <https://www.python.org/downloads/>
```

Comment:
Hm... do we need to specify python? That's already a prerequisite of bootstrap, no?

Reply:
Probably yeah. I'll remove it.

Reply:
Removed.